### PR TITLE
Add blog post "Road to BDK 1.0"

### PR DESCRIPTION
**Pull request by afilini**

### CODE REVIEW

**Comment by danielabrozzoni** [comment]
ACK 08f41c1eea8e74a27c1bcfb0a02125834a8ca741

**Comment by evanlinjin** [comment]
ACK 08f41c1eea8e74a27c1bcfb0a02125834a8ca741

File: docs/_blog/road_to_bdk_1.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Removing many of our current dependencies from the core components of BDK will also allow us to lower our MSRV considerably, which in turn will allow BDK to compile on older distros or entirely different toolchains like
+[`mrustc`], which usually don't keep up with `rustc` in terms of language features.
+
+## Architecture
+
+Roughly speaking, after integrating bdk\_core into BDK the architecture will look like this:
+
+- bdk\_core: this crate will contain all the low level components of a Bitcoin wallet. For example, using this low level API it will be possible to keep track of arbitrary scripts (without the limitations
+  of descriptors) or apply individual blocks to the state of the wallet
```
Comment (RCasatta) [inline docs/_blog/road_to_bdk_1.md:67]: > without the limitation of descriptors

I think this affirmation needs at least a footnote  explaining which are these limitations

Comment (evanlinjin) [reply docs/_blog/road_to_bdk_1.md:67]: *not just scripts derived from the wallet descriptor

Comment (afilini) [reply docs/_blog/road_to_bdk_1.md:67]: Not only that, also scripts that can't be expressed as descriptors

**Pull request merged**